src/SFA.DAS.ApprenticeCommitments/Data/Models/ApprenticeCommitmentsDbContext.cs

```diff
@@ -51,13 +51,33 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<Apprenticeship>()
                 .HasKey(a => a.Id);
 
+            modelBuilder.Entity<Apprenticeship>()
+                .OwnsOne(e => e.Details, details =>
+                {
+                    details.Property(p => p.EmployerAccountLegalEntityId).HasColumnName("EmployerAccountLegalEntityId");
+                    details.Property(p => p.EmployerName).HasColumnName("EmployerName");
+                    details.Property(p => p.TrainingProviderId).HasColumnName("TrainingProviderId");
+                    details.Property(p => p.TrainingProviderName).HasColumnName("TrainingProviderName");
+                    details.OwnsOne(e => e.Course, course =>
+                    {
+                        course.Property(p => p.Name).HasColumnName("CourseName");
+                        course.Property(p => p.Level).HasColumnName("CourseLevel");
+                        course.Property(p => p.Option).HasColumnName("CourseOption");
+                    });
+                });
+
```

Comment:
Nice, first-time I've seen the use of ownsone. Will definitely have conflicts but definitely worthwhile doing this